src/drive/mobile/styles/uploadprogression.styl

```diff
@@ -1,12 +1,18 @@
-@require 'cozy-ui'
+.coz-progress {
+    height: 48px;
+    padding: 5px 16px;
+    box-sizing: border-box;
+    border-bottom: 1px solid #d6d8da;
+    transition: bottom 0.5s ease-out, top 0.5s ease-out, opacity 0.2s ease-out;
+    display: flex;
+}
```

Comment:
We're not using colons and semi-colons elsewhere in the css codebase, so for consistency I think they should be removed here.

Reply:
I don't know what is the stylus syntax, that's such a shame.
But I think you are right.